enterprise/1.5.x/release-notes.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Kong Enterprise 1.5.x Release Notes
+---
+
```

Comment:
This file is sitting in the wrong place in the site directory, creating a new new "enteprise" folder. Should be moved to app/enterprise/1.5.x/release-notes.md

Reply:
Thank you @lena-larionova for fixing the location of this file.